src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java

```diff
@@ -244,7 +244,7 @@ public String[] parseReferences(final String name, final InputStream in, final B
      * @throws Pack200Exception if a problem occurs with an unexpected value or unsupported Codec
      */
     public String[][] parseReferences(final String name, final InputStream in, final BHSDCodec codec,
-        final int counts[], final String[] reference) throws IOException, Pack200Exception {
+                                      final int[] counts, final String[] reference) throws IOException, Pack200Exception {
```

Comment:
Leave this formatting alone, please; it creates too much noise to review in the PR.

Reply:
Done